pkg/beacon/registry/registry.go

```diff
@@ -3,3 +3,5 @@ package registry
 import "github.com/ipfs/go-log"
 
 var logger = log.Logger("keep-registry")
+
+const DefaultStoragePath = ".storage/"
```

Comment:
I wouldn't make it hidden by default. I am afraid this will bring a lot of confusion and problems to node operators. The directory with content that is not visible in `ls` can be deleted by mistake.

```
❯ ls -l keep_data
total 0
❯ ls -la keep_data
total 0
drwxr-xr-x   3 piotr  staff    96 Jul 14 12:05 .
drwxr-xr-x  36 piotr  staff  1152 Jul 14 12:05 ..
drwxr-xr-x   3 piotr  staff    96 Jul 14 12:05 .storage
```